packages/core/test/components/Tabs.test.tsx

```diff
@@ -250,4 +250,22 @@ describe('<Tabs/>', () => {
         .prop('borderless'),
     ).toBe(true);
   });
+
+  it('adds popstate listener when persistWithHash', () => {
```

Comment:
you'll most likely need to add a test for coverage that gets `handlePopstate` to be triggered. check out Dropdown's test `closes when clicked outside` for an example.